### DISCUSSION TITLE
Chance to use extension features/commands as macros

### DISCUSSION THREAD

**Discussion by barux** [alefragnani/vscode-project-manager#732] [Q&A] [answered]
Hello,

I recently purchased a macro keyboard to optimize my workflow and I was wondering if there are specific events or commands provided by the extension itself that I can use to achieve my goal. Specifically, I aim to assign a macro key that opens all projects saved with a specific tag. If possible, I would like to open only some projects with the same tag, perhaps filtered by a boolean value like "enabled."

I can achieve this feature using both VSCode integrated commands and the VSCode CLI from PowerShell.
Thanks in advance for your help.

**Comment by alefragnani** [comment, marked as answer]
Hi @barux ,

I'm not sure how your macro keyboard wors, but maybe #667 would fit your needs, right? If yes, please upvote, so I'm able to identify the most wanted features.

Hope this helps
